cmd/cloud_run.go

```diff
@@ -150,7 +150,7 @@ func (c *cmdCloudRun) flagSet() *pflag.FlagSet {
 		&c.noUsageReport,
 		"no-usage-report",
 		c.noUsageReport,
-		"only when using the local-execution mode, don't send anonymous stats to the developers",
+		"only when using the local-execution mode, don't send anonymous usage stats (https://grafana.com/docs/k6/latest/set-up/usage-collection/)",
```

Comment:
I know the other place it's out of the scope of this PR; but to avoid discrepancies, could you please update the same text in other place there's `--no-usage-report` flag, please?

https://github.com/grafana/k6/blob/master/cmd/config.go#L33 🙏🏻

Reply:
Arf, cheers for pointing it out, I looked for the text pattern but couldn't find it... Addressing it 🙏🏻